src/hooks/useDailyStatistics/utils.ts

```diff
@@ -39,16 +48,12 @@ export const countTotalTransactionsAndByCategory = (
               return key;
             })(),
           category: key,
-          quantityText: formatQuantityText(
-            totalQuantityInCategory,
-            findItemByCategory?.quantity.unit || {
-              type: "POSTFIX",
-              label: " qty",
-            }
-          ),
+          quantity: totalQuantityInCategory,
+          unitType: findItemByCategory?.quantity.unit,
```

Comment:
I think the default fallback value should be here, rather than leaving it as `unitType: undefined`. That doesnt really make sense haha